backend/internal/domain/model/post.go

```diff
@@ -27,3 +27,15 @@ type CreatePostParam struct {
 	Title string `json:"title"`
 	Body  string `json:"body"`
 }
+
+func (post *Post) NewUpdatePost(title, body string) *Post {
```

Comment:
IMO
```suggestion
func (post *Post) UpdatePost(title, body string) *Post {
```
ごめん、さっきコメントしたら良かったけど、これはファクトリじゃないと思うからこっちの方が良いかも？

Reply:
あれ、これ前話してたのか、
じゃあこのままでも良いのかな？
直しても直さなくてもどっちでも良いかも？

Reply:
ファクトリではないから[b4f745a](https://github.com/givery-bootcamp/dena-2024-team3/pull/41/commits/b4f745a02908fecf2d35470c7cda212e58a2f087)で修正しました。